File: src/core/app/app/models/audit.py
from sqlalchemy import BigInteger, Column, DateTime, PrimaryKeyConstraint, Text

from app.db.base_class import FLBase


class Audit(FLBase):
    __tablename__ = "Audits"

    __table_args__ = (PrimaryKeyConstraint("Id", name="idx_20741_PK_Audits"),)

    id = Column("Id", BigInteger, quote=True)
    timestamp = Column("Timestamp", DateTime, nullable=False, quote=True)
    userName = Column("UserName", Text, quote=True)
    tableName = Column("TableName", Text, quote=True)
    action = Column("Action", Text, quote=True)
    keyValues = Column("KeyValues", Text, quote=True)
    oldValues = Column("OldValues", Text, quote=True)
    newValues = Column("NewValues", Text, quote=True)

File: Web/Db/Auditing/Audit.cs
using System;

namespace Web.Db.Auditing
{
    public class Audit  
    {  
        public long Id { get; set; }  
        public DateTimeOffset Timestamp { get; set; }  
        public string Username { get; set; }  
        public string TableName { get; set; }  
        public string Action { get; set; }  
        public string KeyValues { get; set; }  
        public string OldValues { get; set; }  
        public string NewValues { get; set; }  
    }  
}
